src/piece/Queen.hpp

```diff
@@ -1,5 +1,5 @@
-#ifndef ChessPlusPlus_Piece_QueenChessPiece_HeaderPlusPlus
-#define ChessPlusPlus_Piece_QueenChessPiece_HeaderPlusPlus
+#ifndef chesspp_piece_Queen_HeaderPlusPlus
+#define chesspp_piece_Queen_HeaderPlusPlus
```

Comment:
I see you reworked the include guards a little bit. Definitely a lot cleaner and less annoying to write out. Make sure this gets documented in the style guide, if it's not already.